### ISSUE TITLE
UnusedVariable: false-positive with $this in list

### ISSUE THREAD

**Issue by carusogabriel** [slevomat/coding-standard#449]
Code to reproduce:
```php
list($this->foo, $this->bar) = [ // Reports: Unused variable $this.
    $foo->foo, $foo->bar,
];
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/91728b628d8519322883606ff70b5f029edb0ef0
